offerings/components/Offerings.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { connect as connectFela } from 'react-fela'
+import { Field, reduxForm as connectForm, FormSection, FieldArray } from 'redux-form'
+import { flow } from 'lodash'
```

Comment:
my mistake for adding this somewhere before, we should be using [`ramda#pipe`](http://ramdajs.com/docs/#pipe)